### PR TITLE
Restructure application and create first application template

### PR DESCRIPTION
**Pull request by JRodmanYoung**
Since I created a 'templates' folder inside of /scripts and html isn't considered a script, I wondered if maybe there wasn't a better way to structure the application. I moved some things around, hopefully it's not horribly ugly.

I set up the first webpage to render, it has the basic structure it needs. I just need to figure out the javascript now.

Here is a screenshot of my single-page application:

image

### CODE REVIEW

File: application/routes.py [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
 BAD_REQUEST_ERROR = 400
-
-@app.route('/')
-def hello_world():
-    return 'Hello World!'
     
+@app.route('/')
+@app.route('/home')
+def home():
+    users = json.loads(get_persons())
```
Comment (youngjl1) [inline application/routes.py:14]: I love that you're reusing this function here :)

File: application/templates/home.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+				<th>last_name</th>
+				<th>email</th>
+				<th>action</th>
+			<tr>
+			{% for user in users %}
+			<tr>
+				<td>{{ user['first_name'] }}</td>
+				<td>{{ user['last_name'] }}</td>
+				<td>{{ user['email'] }}</td>
+				<td>update</td>
```
Comment (youngjl1) [inline application/templates/home.html]: Per the requirements this should be "delete" not "update." Also needs to be a button but text works for now.

File: application/templates/home.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<html>
+	<head>
+		<title>Welcome to SantaBot</title>
+	</head>
+	
+	<body>
+		<h1>Greetings, Meatbag. Welcome to SantaBot.</h1>
+		<table>
```
Comment (youngjl1) [inline application/templates/home.html:8]: sometimes tables have a additional tags thead and tbody: https://www.w3schools.com/tags/tag_thead.asp. I think they can be useful in distinguishing a tr from a header vs body row.

**Review by youngjl1 — APPROVED** [review]
A couple nits to address and then ready to merge. I love how fast you put this together.

File: application/templates/home.html (hunk 2 above)
Comment (JRodmanYoung) [reply application/templates/home.html]: ok

File: application/templates/home.html (hunk 3 above)
Comment (JRodmanYoung) [reply application/templates/home.html:8]: ok

**Pull request merged**